tyr-core/src/main/java/org/jboss/tyr/model/yaml/LabelFormat.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019-2021 Red Hat, Inc, and individual contributors.
```

Comment:
new classes should have only 2021 as they are new, but I won't block on this. Can be fixed later.